app/middleware/prerender.js

```diff
@@ -1,19 +1,30 @@
+const addAddressLine = require('../lib/addAddressLine');
 const addBankHolidayMessage = require('../lib/addBankHolidayMessage');
 const choicesServices = require('../lib/choicesServices');
 const mapLink = require('../lib/mapLink');
 const updateOpeningTimes = require('../lib/updateOpeningTimes');
 
+function updateServices(searchCriteria, services) {
+  if (services && Array.isArray(services)) {
+    /* eslint-disable no-param-reassign */
```

Comment:
It would be nice to not be manipulating the input in all these functions. Something for a future refactor.

Reply:
Agreed, it's a bit odd modifying the parameter, and returning the value, as it's not obvious there are side affects.

Reply:
Indeed. It does make it more fun trying to reason about the code when you need to have half a dozen different files open!